style.css

```diff
@@ -0,0 +1,127 @@
+@import url('https://fonts.googleapis.com/css2?family=Anton&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Oswald&display=swap');
+
+body{
+  margin: 0;
+  font-family: "Courier New", Courier, monospace;
```

Comment:
Great that you are using a fallback font!

Reply:
!!! :)